packages/shared/src/components/fields/ImageInput.tsx

```diff
@@ -48,7 +48,8 @@ const sizeToIconSize: Record<Size, IconSize> = {
   medium: IconSize.Small,
   large: IconSize.Medium,
 };
-const ACCEPTED_TYPES = 'image/png,image/jpeg';
+export const ACCEPTED_TYPES = 'image/png,image/jpeg';
```

Comment:
gif support will come later?

Reply:
Should be on the same release, let me add it as custom only on the areas where it is needed.